mqtt-common/src/main/java/org/apache/rocketmq/mqtt/common/facade/SubscriptionPersistManager.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.rocketmq.mqtt.common.facade;
```

Comment:
1）better to separate the definition of subscription storage interface and the keeplive recalculation?
2）better to add unit tests to verify these changes

Reply:
Thanks for your advice